lib/ViewModels/TerriaViewer.ts

```diff
@@ -33,7 +35,19 @@ export default class TerriaViewer {
   readonly terria: Terria;
 
   @observable
-  baseMap: MappableMixin.MappableMixin | undefined;
+  private _baseMap: MappableMixin.MappableMixin | undefined;
+
+  get baseMap() {
+    return this._baseMap;
+  }
+
+  async setBaseMap(baseMap?: MappableMixin.MappableMixin) {
+    if (CatalogMemberMixin.isMixedInto(baseMap)) await baseMap.loadMetadata();
+
+    if (baseMap) await baseMap.loadMapItems();
+
+    runInAction(() => (this._baseMap = baseMap));
+  }
```

Comment:
Why?

Reply:
Sorry I forgot to make a change to the new https://github.com/TerriaJS/terriajs/blob/53275616ff28f74d4b1be02ff8ec65d45b2153cf/lib/Models/BaseMaps/BaseMapModel.ts file.

I just added https://github.com/TerriaJS/terriajs/commit/53275616ff28f74d4b1be02ff8ec65d45b2153cf

This is so basemaps are loaded when they are selected - instead of all on startup